### ISSUE TITLE
Role Detail Views - Misc

### ISSUE THREAD

**Issue by DarksightKellar** [decentdao/decent-interface#2582]
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#14d2ee9c5bc280e9a264c3fd107623d7

The core ask in this task, while completely valid, is ill-timed. Trying to squeeze the work in before the overhaul effort being planned by @decentdao/engineering will lead to wasted effort, as it requires significant redesign and decision making on both UI and technical side.

@Da-Colon can you plz edit this description with the compromise settled on on standup today?
cc @nicolaus-sherrill

**Comment by Da-Colon** [comment]
I've had a draft in here all day but its quite long to explain the conversation that was sparked but the initial concern @DarksightKellar brought up. 

To save for a chance to elaborate, the TL:DR to kinda 'eliminate' the jarring effect of a separate right drawer opening on navigating to 'edit, We talked about just moving the form into a `Modal`. 

There was discussion of 'edit in place', I have some push back I'd like to discuss as I believe this complicates future efforts:
- Member address is also copiable here?
- Edit in place, breaks other patterns in app for users changing/editing.
- For Introducing a global proposal builder and 'add role' action, we would need these forms to be in a mobile anyway. (This is where we kept going with the conversation)

@parkermccurley
